source/features/index.tsx

```diff
@@ -77,7 +77,10 @@ const logError = (url: string, error: unknown): void => {
 		\`\`\`
 	`));
 	newIssueUrl.searchParams.set('example_url', location.href);
-	newIssueUrl.searchParams.set('browser_s_used', navigator.userAgent);
+	// @ts-expect-error navigator.userAgentData is not typed
+	const {platform} = navigator.userAgentData || navigator;
+	const browser = /(Chrome|Firefox|Safari)\/\d+/.exec(navigator.userAgent)?.[0];
+	newIssueUrl.searchParams.set('browser_s_used', [platform, browser].join(' '));
```

Comment:
Not jumping into the rabbit hole but this should be enough (platform/OS name + browser name + major browser version).

"Chrome" here matches any Chromium-based browser, since we only care about _its_ version.

As an example, it produces `macOS Chrome/96`, `MacIntel Firefox/94` and `MacIntel Safari/605` on my machine.